improver_tests/acceptance/test_apply_emos_coefficients.py

```diff
@@ -79,6 +79,46 @@ def test_truncated_normal(tmp_path):
     acc.compare(output_path, kgo_path, atol=LOOSE_TOLERANCE)
 
 
+def test_normal_each_point_sites(tmp_path):
+    """Test using a normal distribution when coefficients have been calculated
+    independently at each site (initial guess and minimisation)."""
+    kgo_dir = acc.kgo_root() / "apply-emos-coefficients/sites/each_point"
+    kgo_path = kgo_dir / "kgo.nc"
+    input_path = kgo_dir / ".." / "input.nc"
+    emos_est_path = kgo_dir / "coefficients.nc"
+    output_path = tmp_path / "output.nc"
+    args = [
+        input_path,
+        emos_est_path,
+        "--random-seed",
+        "0",
+        "--output",
+        output_path,
+    ]
+    run_cli(args)
+    acc.compare(output_path, kgo_path, atol=LOOSE_TOLERANCE)
+
+
+def test_normal_minimise_each_point_sites(tmp_path):
```

Comment:
For apply_emos, we only really need a test for coefficients at each point in addition to the existing tests for coefficients for the whole domain. I don't think we need a test for both --each-point and --minimise-each-point here.

Reply:
I've removed this test.